alvr/session/src/settings.rs

```diff
@@ -671,6 +671,9 @@ pub enum BodyTrackingSinkConfig {
 pub struct BodyTrackingConfig {
     pub sources: BodyTrackingSourcesConfig,
     pub sink: BodyTrackingSinkConfig,
```

Comment:
Techincally this should be "steamvr-restart", i missed it in the previous PR.

Reply:
the switch itself is already marked that